### DISCUSSION TITLE
Welcome to Xiaomi Home Discussions!

### DISCUSSION THREAD

**Discussion by SusanPhevos** [XiaoMi/ha_xiaomi_home#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share ideas.
  * Give suggestions.
  * Ask questions you’re wondering about.
  * Answer questions if you can.
  
If you come across any bugs, you can report them by [opening an issue](https://github.com/XiaoMi/ha_xiaomi_home/issues/new/choose). We will try to resolve the issue 💪.

In short, this is an open discussion area where everyone is welcome to speak up, share ideas and experiences. Thank you for your support!

---

## 👋 欢迎来到讨论区！

这里是一个为大家提供交流的平台。您可以为我们的产品提供建议，提出您在使用产品时遇到的问题，或是解答其他用户的疑惑。我们希望这里能够成为一个充满活力的社区，让我们一起打造一个更好的产品。

如果您遇到了 bug，您可以通过[创建议题](https://github.com/XiaoMi/ha_xiaomi_home/issues/new/choose)来报告问题。我们会尽快响应并努力解决问题 💪。

总之，这里是一个开放的讨论区，欢迎大家踊跃发言，分享想法和经验。感谢您的支持！

**Comment by alamibug** [comment]
nb

**Comment by PaulChen0127** [comment]
我的米家app中绑定了好太太智联（设备未在米家app界面呈现卡面形式）、日立中央空调（设备已在米家app界面呈现卡面形式），这两种情况的设备能否同步至HA中？

**Reply by PaulChen0127** [reply]
补充：Xiaomi Miot Auto在之前可以讲将以绑定的日立空调显示在HA中并可用，现在不行了。希望xiaomi官方能支持，这会方便很多。

**Comment by wessamlauf** [comment]
Hi, I am using Xiaomi C200 camera, I can see the device in HA via Xiaomi Home integration but i am not able to see the stream, is it possible ?

**Comment by zhangkaiyv** [comment]
建议米家智能小饭煲2chunmi.cooker.c301加入扩展模式
![IMG_0437](https://github.com/user-attachments/assets/e88cd53f-58cf-4aba-834f-23b9fc9b0301)
